packages/wrangler/src/index.tsx

```diff
@@ -628,21 +628,23 @@ function createCLIParser(argv: string[]) {
         isWritingScripts: boolean,
         isCreatingWranglerToml: boolean,
         packagePath: string,
-        scriptPath: string
+        scriptPath: string,
+        extraToml: TOML.JsonMap
       ) {
         if (isCreatingWranglerToml) {
-          // rewrite wrangler.toml with main = "path/to/script"
+          // rewrite wrangler.toml with main = "path/to/script" and any additional config specified in `extraToml`
           const parsedWranglerToml = parseTOML(
             readFileSync(wranglerTomlDestination)
           );
-          fs.writeFileSync(
-            wranglerTomlDestination,
-            TOML.stringify({
+          const newToml = {
+            ...{
```

Comment:
I don't think you need to spread over these properties...

```ts
const newToml = {
  name: parsedWranglerToml.name,
  main: scriptPath,
  compatibility_date: parsedWranglerToml.compatibility_date,
  ...extraToml,
};

Reply:
Ok, I'll update it as your comment.